stories/Map.tsx

```diff
@@ -26,11 +30,27 @@ storiesOf('Atoms|Map', module)
                 },
                 radius: 30000,
             };
+            const pointMarker = {
+                center: {
+                    lat: 52.5112671,
+                    lng: 7.2535521,
+                },
+            };
             const markers = {
                 marker1: [defaultMarker],
                 marker2: [addedMarker],
                 multiple: [defaultMarker, addedMarker],
                 none: [],
+                'pointer marker': [pointMarker],
+                areas: [DR_DRENTHE, FL_FLEVOLAND, FR_FRIESLAND, GE_GELDERLAND],
```

Comment:
indeed this takes visibly long time. Can we just list here, and the next example, just 1 area. I think that demonstrates the component well enough. It is not OneUI's responsibility to optimize the performance caused by large data passed as prop. So it is ok, if we simplify it here

Reply:
Left only one geoJson object.